core/src/migrations/mysql/1609105692434-AddPronunciationField.ts

```diff
@@ -0,0 +1,22 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class AddPronunciationField1609105692434 implements MigrationInterface {
+    name = 'AddPronunciationField1609105692434'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query("ALTER TABLE `dictionary_entry` ADD `pronunciation` varchar(255) NULL");
+        await queryRunner.query("ALTER TABLE `bible_version` DROP COLUMN `abbreviation`");
```

Comment:
this deletes existing data. i think this and the following lines can be removed

Reply:
done